docs/docs/main/extensions/create.md

```diff
@@ -41,13 +43,134 @@ To create an extension, you need to opt-in to `ExperimentalKatalogExtApi`.
 
 To use the extension, specify it when you call the `registerKatalog` method.
 
-```kotlin
+```kotlin {4}
 registerKatalog(
     title = "Android Sample",
     extensions = listOf(
-        createSampleExt()
+        createSampleExt() // append this line
```

Comment:
```suggestion
        createSampleExt() // add this line
```

`append` means "to add something to the end of a piece of writing", so I'd use `add` here.